CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### 1.17.0
+- Feat: Supported docker arm64 image [#1027](https://github.com/redhat-developer/yaml-language-server/pull/1027)
+- Fix: Reverted ajv-draft04 as it not support HTTPS [#1026](https://github.com/redhat-developer/yaml-language-server/pull/1026)
+- Fix: Use replaceAll() instead of replace() when turning a label into a regex [#1078](https://github.com/redhat-developer/vscode-yaml/issues/1078)
```

Comment:
this is only in vscode yaml, not the language server, so I think you shuold remove it from here

Reply:
No we usually added both. As there were no sperate release for YAML LS

Reply:
sounds good my bad